### ISSUE TITLE
Outdated?

### ISSUE THREAD

**Issue by gityaks** [Mimerme/Yak-Hax#1]
I am trying to get data from yik yak for analysis. Is this repository outdated, because the yikyak api seems to have vanished.

**Comment by Mimerme** [comment]
Yes, this could very much be outdated. I haven't used it personally for a long time. I could always help you pinpoint the issue. Also check out the wiki. You may need to change the YikYak Version string to satisfy YikYak's desire to force users to update.

**Comment by NSExceptional** [comment]
Hey Mimerme, I have a few questions for you about the API! I'm writing my own wrapper. Do you have iMessage or something where we could talk?
